invesalius/navigation/navigation.py

```diff
@@ -151,6 +151,7 @@ def __init__(self, pedal_connection, neuronavigation_api):
         self.obj_reg = None
         self.track_obj = False
         self.m_change = None
+        self.obj_data = None
```

Comment:
I checked and the obj_data does not need to be a self. variable. It is passed directly as an argument inside the coreg_data and is not used in other methods inside the class. Why made self. was added? Could it be removed?

Reply:
I had to make obj_data as a self variable to be available to other functions. Im using obj_data in task_navigator
`m_target = dcr.image_to_tracker(self.navigation.m_change, coord_raw, nav_target, self.icp, self.navigation.obj_data)`